content/pt-br/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Criando Pods Estáticos
+weight: 170
+content_type: task
+reviewers:
+- jsafrane
+---
+
+<!-- overview -->
+
+
+*Pods Estáticos* são gerenciados diretamente pelo `daemon` kubelet em um nó específico,
+sem o {{< glossary_tooltip text="servidor de API" term_id="kube-apiserver" >}}
+observando-os.
+Ao contrário dos pods que são gerenciados pelo `Control Plane` (por exemplo, uma
+{{< glossary_tooltip text="Implantação" term_id="deployment" >}});
+em vez disso, o kubelet observa cada Pod estático 
+(e reinicia-os se falharem).
+
+Pods estáticos estão sempre ligados a um {{< glossary_tooltip term_id="kubelet" >}} em um nó específico.
+
+O Kubelet tenta automaticamente criar um {{< glossary_tooltip text="mirror Pod" term_id="mirror-pod" >}}
+no servidor de API do Kubernetes para cada Pod estático.
+Isso significa que os pods em execução em um nó são visíveis no servidor de API,
+mas não podem ser controlados a partir daí.
+Aos nomes de Pods será acrescido um sufixo com o nome de host do nó, com um hífem. 
```

Comment:
```suggestion
Aos nomes de Pods será sufixados com o nome de host do nó, com um hífem à esquerda. 
```
I looked at the original documentation and I think this is better and the"sufixado" exists, we can use it here